js/nivel2.js

```diff
@@ -0,0 +1,351 @@
+
+    Ejemplo:
+
+    // punto de partida
+    let numero = 0;
+
+    while (numero < 10) {
+      // 'menos de 10' es una condición (punto final)
+
+      console.log('JavaScript es increíble!');
+
+      numero = numero + 1;
+      // + 1 es un contador / tamaño del paso
+    }
+*/
+
+// TODO: Usando un bucle 'while', decile a tu computadora que registre los números de
+//       diez a uno.
+
+
+
+
+
+
+/*
+    Bucles for
+    ==========
+
+    Los bucles for son muy similares al 'bucle while'. En un bucle for
+    se declara un contador en la declaración.
+
+    Ejemplo:
+
+    let i;
+    for (i = 0; i <= 5; i = i + 1) {  // (punto de partida; condicion; paso)
+      console.log('Purr');
+    }
+*/
+
+// TODO: Registre cada 3er número del 3 al 22 usando un 'bucle for'.
+
+
+
+
+
+
+/*
+    Iterando a traves de Arrays
+    ===========================
+
+    Ahora que conocemos los bucles, vamos a usar cada valor de mi lista de animales
+    y expresar mi amor por cada uno ¿Cómo lo voy a hacer?
+
+    Podemos usar un 'bucle for' para iterar a través de nuestro array y obtener cada valor
+    de eso.
+
+    Nota: i++ es otra forma de expresar i = i + 1.
+
+    Ejemplo:
+
+    const animales = ['gatos', 'perros', 'caballos'];
+
+    for(let i = 0; i < animales.length; i++){
+        console.log('Me gustan los ' + animales[i]);
+    }
+*/
+
+// TODO: Probalo con tu comidasFavoritas array.
+
+
+
+
+
+
+/*
+    Bucles y Logica
+    ===============
+    
+    Traigamos bucles junto con las declaraciones if / else que aprendimos en
+    nivel 1, y hagamos algo interesante.
+
+
+    Contemos de 10 a 0 e imprimamos todos los números. Pero cuando lleguemos al
+    medio (5) imprimimos 'Woohoo, estamos en el medio!'.
+
+    Ejemplo:
+
+    for (let i = 10; i >= 0; i = i - 1) {
+        if (i === 5) {
+            console.log('WooHoo, estamos en el medio!');
+        } else {
+            console.log(i);
+        }
+    }
+*/
+
+// TODO: Ha llegado el momento de un ejercicio clásico: 'FizzBuzz'.
+
+// Cuenta del 1 al 50 e imprime los números:
+//
+// * Si un número es múltiplo de tres, imprime 'Fizz'.
+// * Si es un múltiplo de 5, imprime 'Buzz'.
+// * Si es un múltiplo de 3 y 5, imprime 'FizzBuzz'.
+// * Para todo lo demás, imprime el número mismo.
+
+// P.S. Es posible que desees utilizar el operador aritmético modulo (%):
+// Calcula el resto al dividir.
+//
+// 10% 3 = 1 - en 10 tenemos 3 * 3 + 1
+// 16% 4 = 0 - en 16 tenemos 4 * 4
+// 19% 4 = 3 - en 19 tenemos 4 * 4 + 3, etc.
+
+
+
+
+
+
+//////////////////////////////////////////////////////////////////////////////
+// ¡Felicidades! ¡Has terminado el Nivel 2 de JavaScript Básico!            //
+// Levántate, estira las piernas y celebra tu logro.                        //
+// El siguiente paso será seguir las instrucciones en el archivo level3.js. //
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
```suggestion
//////////////////////////////////////////////////////////////////////////////

```

Una línea vacía al final del archivo :)